uiplib/utils.py

```diff
@@ -17,6 +19,20 @@ def make_dir(dirpath):
     if sys.platform.startswith('linux'):
         os.chmod(dirpath, 0o777)
 
+
+def exit_UIP(service=False):
+    pid_file = os.path.join(HOME_DIR, 'daemon-uip.pid')
+    if service:
```

Comment:
No, we actually do.
The thing is that if we do `UIP --service stop` when the service is not running then,  we need to print `You need to start UIP as a service first`
Thats, why you require that if and else.